Apps/WebClient/src/Server/Models/AcaPy/Connection.cs

```diff
@@ -0,0 +1,26 @@
+//-------------------------------------------------------------------------
+// Copyright © 2019 Province of British Columbia
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//-------------------------------------------------------------------------
+namespace HealthGateway.WebClient.Server.Models.AcaPy
```

Comment:
I dont think the ```.Server``` section is needed since it is a server already

Reply:
The .server should not be present but VS automatically added it and I missed it.  I'll remove it.